odl/discr/partition.py

```diff
@@ -546,6 +555,139 @@ def squeeze(self):
         newset = self.set[self.grid.inondeg]
         return RectPartition(newset, self.grid.squeeze())
 
+    def index(self, value, floating=False):
+        """Return the index of a value in the domain.
+
+        Parameters
+        ----------
+        value : ``self.set`` element
+            The point to find the index of.
+        floating : bool
+            True if the index should also give the position inside the voxel.
+            This is given by returning the integer valued index of the voxel
+            plus the partial distance from the left to the right boundary.
```

Comment:
Almost perfect, just at the end it's better to speak of "distance from the left cell boundary" only, since the distance from the left to the right boundary is the cell size by definition. Perhaps one could be even clearer by adding "in terms of multiples of the cell sizes" or something like that, to make clear that it's not an absolute quantity.